mhkit/wave/io.py

```diff
@@ -112,30 +112,32 @@ def read_NDBC_file(file_name, missing_values=['MM',9999,999,99]):
 
 
 def ndbc_available_data(parameter='swden',
-                        number=None, 
+                        buoy_number=None, 
                         proxy=None):  
     '''
-    For a given parameter this will return a DataFrame of years, and 
-    station IDs and file names              
+    For a given parameter this will return a DataFrame of years, 
+    station IDs and file names that contain that parameter data.              
     
     Parameters
     ----------
-    parameter: string
-        'swden'	:	'Raw Spectral Wave Current Year Historical Data'
-    number: string
+    parameter: string (optional)
+        Current option is 'swden'	:	'Raw Spectral Wave Current Year Historical Data'
+        More parameter options will be available soon. 
```

Comment:
same as the previous comment

Reply:
@ssolson my response to all of these comments is to ask how long you think it will be until you have the capability to add more parameter options? If this is something that can be accomplished fairly soon (month to 3 months), I'd say let's hold off releasing this code until more options are available. If you want to release this code, I would advise temporarily removing the "parameter" input and just have 'swden' hard coded at the top of the function. It would not be too heavy of a lift to re-introduce "parameter" as an input option down the road.